server/models/social_account.ts

```diff
@@ -33,6 +34,7 @@ export default (
     provider: { type: dataTypes.STRING },
     provider_username: { type: dataTypes.STRING },
     provider_userid: { type: dataTypes.STRING },
+    user_id: { type: dataTypes.STRING },
```

Comment:
Do we need to add an association for this? Not sure what `user_id` explicitly refers to.

Reply:
As I understood it, it's the user_id associated with the social account. I could have misinterpreted this. I use it [here](https://github.com/hicommonwealth/commonwealth/blob/5c84e6c9d5f9346c7edda27d597c6e181134efe1/server/routes/getNewProfile.ts#L68).